### ISSUE TITLE
How long does it normally take to train the sample data

### ISSUE THREAD

**Issue by Graystripe17** [llSourcell/tensorflow_chatbot#25]
Sorry this is a basic question. I have 2.5 GHz Intel i7 quad core on Mac and when I ran execute.py to train, it heated my CPUs to 90 Celsius so I had to shut it off after 5 minutes because I was worried about heat damage. I am running the tensorflow CPU only installation.

Here is where I stopped it 

global step 300 learning rate 0.5000 step-time 1.42 perplexity 455.42
  eval: bucket 0 perplexity 138.66
  eval: bucket 1 perplexity 202.11
  eval: bucket 2 perplexity 235.91
  eval: bucket 3 perplexity 282.56

It was using all of my RAM and I was wondering if I did something wrong. Should I use the GPU installation? How long does it normally take? I don't want to risk overheating. Thanks!

**Comment by Graystripe17** [comment]
I am currently on global step 1800 after an hour
